@ndlib/gatsby-theme-marble/src/components/Shared/NDBrand/Logos/ClickableRBSCLogoWhite/index.js

```diff
@@ -6,31 +6,44 @@ import { jsx, Image } from 'theme-ui'
 import ndLogo from '@ndlib/gatsby-theme-marble/src/assets/logos/hesburgh_mark_H2_white.svg'
 import rbsc from '@ndlib/gatsby-theme-marble/src/assets/logos/rbsc.svg'
 
-const ClickableRBSCLogoWhite = ({ variant, width, height, rbscWidthHeight }) => {
+const ClickableRBSCLogoWhite = ({ variant }) => {
   return (
     <div sx={{ variant: variant }}>
       <a href='https://www.library.nd.edu/' >
-        <Image src={ndLogo} width={width} height={height} alt='Hesburgh Libraries, University of Notre Dame' sx={{ marginBottom: '0.5rem !important' }} />
+        <Image
```

Comment:
The reason height and width stayed in this call was because lighthouse would complain about not having a width or height on the SVG.